rholang/src/main/scala/coop/rchain/rholang/interpreter/RholangCLI.scala

```diff
@@ -109,7 +109,12 @@ object RholangCLI {
             val evaluatorFuture = evaluate(runtime.reducer, par).runAsync
             waitThenPrintStorageContents(evaluatorFuture, runtime.store)
           case Left(error) =>
-            error.printStackTrace(Console.err)
+            // we don't want to print stack trace for syntax errors
+            if (error.getMessage.toLowerCase.contains("syntax")) {
```

Comment:
We should use a particular Error type rather than a string type, and then case match.

Reply:
The error thrown is just a string wrapped in generic `Error` thrown by the BNFC-generated code. I don't know whether we should match on the generic type.